message/pb/message.proto

```diff
@@ -32,6 +33,9 @@ message Message {
   enum BlockPresenceType {
     Have = 0;
     DontHave = 1;
+    Unknown = 3;
```

Comment:
Sometimes we're going to need a response that indicates "I'm not giving you an answer" rather than "I have the data, but won't give it to you because you're not authorized"